product_docs/docs/pgd/5/node_management/subscriber_only.mdx

```diff
@@ -6,39 +6,39 @@ title: Subscriber-only nodes and groups
 
 As the name suggests, a subscriber-only node subscribes only to replication
 changes from other nodes in the cluster. However, no other nodes receive
-replication changes from `subscriber-only` nodes. This is somewhat similar to
-logical standby nodes. But in contrast to logical standby, the `subscriber-only`
+replication changes from subscriber-only nodes. This is somewhat similar to
+logical standby nodes. But in contrast to logical standby, the subscriber-only
 nodes are fully joined to the cluster. They can receive replication changes from
 all other nodes in the cluster and hence aren't affected by unavailability or
 parting of any one node in the cluster.
 
-A `subscriber-only` node is a fully joined PGD node and hence it receives all
+A subscriber-only node is a fully joined PGD node and hence it receives all
 replicated DDLs and acts on those. It also uses Raft to consistently report its
-status to all nodes in the cluster. The `subscriber-only` node doesn't have Raft
+status to all nodes in the cluster. The subscriber-only node doesn't have Raft
 voting rights and hence can't become a Raft leader or participate in the leader
 election. Also, while it receives replicated DDLs, it doesn't participate in DDL
-or DML lock acquisition. In other words, a currently down `subscriber-only` node
+or DML lock acquisition. In other words, a currently down subscriber-only node
 doesn't stop a DML lock from being acquired.
 
-The `subscriber-only` node forms the building block for PGD Tree topology. In
+The subscriber-only node forms the building block for PGD Tree topology. In
```

Comment:
Wanted to check on capitalization or the word "tree." Can you clarify why this might be a proper noun?

Reply:
I believe it could be lower-cased.
```suggestion
The subscriber-only node forms the building block for PGD tree topology. In
```